krkn/prometheus/collector.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import annotations
+
+import datetime
+import logging
+from typing import Dict, Any, List, Optional
+
+import yaml
+from krkn_lib.prometheus.krkn_prometheus import KrknPrometheus
+from krkn.prometheus.client import metrics as client_metrics
+
+
+def collect_prometheus_metrics(
+    prom_cli: KrknPrometheus,
+    metrics_profile: str | dict,
+    start_time: datetime.datetime,
+    end_time: datetime.datetime,
+    granularity: int = 30,
+) -> list[dict[str, Any]]:
+    """Collect Prometheus metrics described by a metrics profile for a given time window.
+
+    Args:
```

Comment:
do we need this function at all since its just calling a pre-existing function? this seems not needed